src/sentry/utils/email.py

```diff
@@ -446,17 +446,27 @@ def get_connection(fail_silently=False):
     )
 
 
-def send_mail(subject, message, from_email, recipient_list, fail_silently=False, **kwargs):
+def send_mail(
+    subject, message, from_email, recipient_list, fail_silently=False, headers=None, **kwargs
+):
     """
     Wrapper that forces sending mail through our connection.
     Uses EmailMessage class which has more options than the simple send_mail
     """
+    if headers is None:
+        headers = {
+            "X-Auto-Response-Suppress": "All",
+            "Precedence": "bulk",
+            "Auto-Submitted": "auto-generated",
+        }
```

Comment:
@DeltaXWizard can you add a test for this change?

Reply:
I don't mind, but I'm not really sure how to write a test using sentry's test utilities with mail headers nor where to start.

Reply:
@DeltaXWizard You can add a unit test here as a starting point: https://github.com/getsentry/sentry/blob/682684dfb1c7cf6090c753dfff592a4a0d3945be/tests/sentry/utils/email/tests.py#L357-L373